modules/porous_flow/test/tests/hysteresis/tests

```diff
@@ -113,6 +113,24 @@
     design = 'PorousFlow1PhaseHysP.md'
     issues = '#16129'
   [../]
+  [./except14]
+    type = 'RunException'
+    input = 'except14.i'
+    expect_err = 'The Dictator announces that the number of phases is 1 whereas PorousFlow2PhaseHysPP can only be used for 2-phase simulation.  When you have efficient government, you have dictatorship.'
+    match_literal = true
+    requirement = 'An exception shall be thrown if a two-phase PP hysteretic capillary-pressure Material is used in models with not 2 phases'
```

Comment:
I'm loving the dictators error messages here, but not the *used in models with not 2 phases*. Maybe *used in models with a different number of phases than two*? Or *used in non-2 phase models*?

Reply:
Actually, that second one doesn't sound right either.

Reply:
oh dear, i feel like computer-language is destroying any good grammar that i might have had.